### PR TITLE
[zh] update outdate setup-konnectivity file

### PR DESCRIPTION
**Pull request by TinySong**
update outdate file

### CODE REVIEW

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 1]
```diff
@@ -32,15 +32,8 @@ You need to configure the API Server to use the Konnectivity service
 and direct the network traffic to the cluster nodes:
 
 1. Make sure that
-the `ServiceAccountTokenVolumeProjection` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:35]: Please run the following command in your local clone, under the **main** branch:

```
./scripts/lsync.sh content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md
```

You will see that line 16 in the English content has changed and a paragraph has been added.

Comment (TinySong) [reply content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:35]: @tengqm  this command is very useful, thanks for your advice.  PR update.

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 2]
```diff
@@ -13,7 +13,15 @@ Konnectivity 服务为控制平面提供集群通信的 TCP 级别代理。
...
 
-{{< include "task-tutorial-prereqs.md" >}}
+<!--
+You need to have a Kubernetes cluster, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this
+tutorial on a cluster with at least two nodes that are not acting as control
+plane hosts. If you do not already have a cluster, you can create one by using
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/).
+-->
+你需要有一个Kubernetes集群，并且kubectl命令可以与集群通信。建议在至少有两个不充当控制平面主机的节点的集群上运行本教程。如果您还没有集群，可以使用
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:23]: 中英文之间要留白（一个空格）。

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 3]
```diff
@@ -13,7 +13,15 @@ Konnectivity 服务为控制平面提供集群通信的 TCP 级别代理。
...
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)创建一个集群。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:24]:
```suggestion
[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) 创建一个集群。
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 4]
```diff
@@ -61,16 +62,8 @@ your API Server egress configuration file.
...
-   [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)
-   被启用。你可以通过为 kube-apiserver 提供以下标志启用
-   [服务账号令牌卷保护](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)：
-
-   ```
-   --service-account-issuer=api
-   --service-account-signing-key-file=/etc/kubernetes/pki/sa.key
-   --api-audiences=system:konnectivity-server
-   ```
+1. [ ] 确保 [Service Account Token Volume Projection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:65]:
```suggestion
确保[服务账号令牌卷投射](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 5]
```diff
@@ -61,16 +62,8 @@ your API Server egress configuration file.
...
+   功能被启用。自 Kubernetes v1.20 起默认已被启用。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:66]:
```suggestion
   特性被启用。该特性自 Kubernetes v1.20 起默认已被启用。
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:23]:
```suggestion
你需要有一个 Kubernetes 集群，并且 kubectl 命令可以与集群通信。
建议在至少有两个不充当控制平面主机的节点的集群上运行本教程。
如果你还没有集群，可以使用
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 6]
```diff
@@ -13,7 +13,15 @@ Konnectivity 服务为控制平面提供集群通信的 TCP 级别代理。
...
 
-{{< include "task-tutorial-prereqs.md" >}}
+<!--
+You need to have a Kubernetes cluster, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this
+tutorial on a cluster with at least two nodes that are not acting as control
+plane hosts. If you do not already have a cluster, you can create one by using
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/).
+-->
+你需要有一个 Kubernetes 集群，并且 kubectl 命令可以与集群通信。建议在至少有两个不充当控制平面主机的节点的集群上运行本教程。如果您还没有集群，可以使用
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:23]: 长行折行；用你不用您。

```suggestion
你需要有一个 Kubernetes 集群，并且 kubectl 命令可以与集群通信。
建议在至少有两个不充当控制平面主机的节点的集群上运行本教程。
如果你还没有集群，可以使用
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 7]
```diff
@@ -61,16 +62,8 @@ your API Server egress configuration file.
...
-   [特性门控](/zh/docs/reference/command-line-tools-reference/feature-gates/)
-   被启用。你可以通过为 kube-apiserver 提供以下标志启用
-   [服务账号令牌卷保护](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)：
-
-   ```
-   --service-account-issuer=api
-   --service-account-signing-key-file=/etc/kubernetes/pki/sa.key
-   --api-audiences=system:konnectivity-server
-   ```
+确保 [服务账号令牌卷投射](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:65]:
```suggestion
确保[服务账号令牌卷投射](/zh/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
```

File: content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 8]
```diff
@@ -61,16 +62,8 @@ your API Server egress configuration file.
...
+   特性被启用。该特性自 Kubernetes v1.20 起默认已被启用。
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubernetes/setup-konnectivity.md:66]:
```suggestion
特性被启用。该特性自 Kubernetes v1.20 起默认已被启用。
```

**Pull request merged**